packages/comparisons/src/data.json

```
@@ -12516,6 +12520,33 @@
 			"status": "skipped"
 		}
 	},
+	{
+		"eslint": [
+			{
+				"name": "@typescript-eslint/member-ordering",
+				"url": "https://typescript-eslint.io/rules/member-ordering"
+			}
+		],
+		"flint": {
+			"name": "memberOrderings",
+			"plugin": "ts",
+			"status": "skipped"
+		},
+		"notes": "Superseded by sorting plugin rules."
+	},
+	{
+		"eslint": [
+			{
+				"name": "@typescript-eslint/method-signature-style",
+				"url": "https://typescript-eslint.io/rules/method-signature-style"
+			}
+		],
+		"flint": {
+			"name": "methodSignatureStyle",
```

I've been going with plurals whenever possible:

```suggestion
			"name": "methodSignatureStyles",
```